packages/edit-site/src/components/dataviews/filters.js

```diff
@@ -49,26 +49,24 @@ export default function Filters( { fields, view, onChangeView } ) {
 		} );
 	} );
 
-	return (
-		view.visibleFilters?.map( ( filterName ) => {
-			const filter = filterIndex[ filterName ];
+	return view.visibleFilters?.map( ( filterName ) => {
+		const filter = filterIndex[ filterName ];
 
-			if ( ! filter ) {
-				return null;
-			}
+		if ( ! filter ) {
+			return null;
+		}
 
-			if ( filter.type === 'enumeration' ) {
-				return (
-					<InFilter
-						key={ filterName }
-						filter={ filter }
-						view={ view }
-						onChangeView={ onChangeView }
-					/>
-				);
-			}
+		if ( filter.type === 'enumeration' ) {
+			return (
+				<InFilter
+					key={ filterName }
+					filter={ filter }
+					view={ view }
+					onChangeView={ onChangeView }
+				/>
+			);
+		}
 
-			return null;
-		} ) || __( 'No filters available' )
```

Comment:
There is a lot of whitespace changes, but the actual change is removing `__( 'No filters available' )`